### ISSUE TITLE
Vertical app grid

### ISSUE THREAD

**Issue by no-comma** [RensAlthuis/vertical-overview#21]
Since the workspaces are vertical the app grid should be too.

**Comment by RensAlthuis** [comment]
I agree, though that might take a while. I doubt it will be an easy thing to change.

**Comment by P4prik4** [comment]
imo just disable it for the application grid.
